lib/iris/tests/integration/test_netcdf.py

```diff
@@ -23,13 +23,15 @@
 # importing anything else.
 import iris.tests as tests
 
+from itertools import repeat
```

Comment:
Yeah, I'm working on a fix for #2195 :scream: 

But I'll kept an eye on your travis jobs and keep restarting them manually. We'll get there ...